v2/tools/generator/internal/codegen/pipeline/exported_configmaps.go

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package pipeline
+
+import "github.com/Azure/azure-service-operator/v2/tools/generator/internal/astmodel"
+
+// ExportedProperties is a map of string property name to a chain of properties. For allowing the export of a property "foo" that writes
+// the status.a.b.c property to a config map would be "foo" -> []{status, a, b, c}
+type ExportedProperties = map[string][]*astmodel.PropertyDefinition
```

Comment:
This type name doesn't actually match the file name. I think I need to rename file.